wasmcloud_host/assets/vendor/wasmcloud/js/wasmcloud.js

```diff
@@ -1,173 +1,173 @@
+//         }]
+//     },
+//     options: {
+//         maintainAspectRatio: false,
+//         legend: {
+//             display: false
+//         },
+//         scales: {
+//             xAxes: [{
+//                 display: false
+//             }],
+//             yAxes: [{
+//                 display: false
+//             }]
+//         },
+//         elements: {
+//             line: {
+//                 borderWidth: 2
+//             },
+//             point: {
+//                 radius: 0,
+//                 hitRadius: 10,
+//                 hoverRadius: 4
+//             }
+//         }
+//     }
+// }); // eslint-disable-next-line no-unused-vars
 
-var cardChart4 = new Chart(document.getElementById('card-chart4'), {
-    type: 'bar',
-    data: {
-        labels: ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December', 'January', 'February', 'March', 'April'],
-        datasets: [{
-            label: 'My First dataset',
-            backgroundColor: 'rgba(255,255,255,.2)',
-            borderColor: 'rgba(255,255,255,.55)',
-            data: [78, 81, 80, 45, 34, 12, 40, 85, 65, 23, 12, 98, 34, 84, 67, 82],
-            barPercentage: 0.6
-        }]
-    },
-    options: {
-        maintainAspectRatio: false,
-        legend: {
-            display: false
-        },
-        scales: {
-            xAxes: [{
-                display: false
-            }],
-            yAxes: [{
-                display: false
-            }]
-        }
-    }
-}); // Random Numbers
+// var cardChart4 = new Chart(document.getElementById('card-chart4'), {
+//     type: 'bar',
+//     data: {
+//         labels: ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December', 'January', 'February', 'March', 'April'],
+//         datasets: [{
+//             label: 'My First dataset',
+//             backgroundColor: 'rgba(255,255,255,.2)',
+//             borderColor: 'rgba(255,255,255,.55)',
+//             data: [78, 81, 80, 45, 34, 12, 40, 85, 65, 23, 12, 98, 34, 84, 67, 82],
+//             barPercentage: 0.6
+//         }]
+//     },
+//     options: {
+//         maintainAspectRatio: false,
+//         legend: {
+//             display: false
+//         },
+//         scales: {
+//             xAxes: [{
+//                 display: false
+//             }],
+//             yAxes: [{
+//                 display: false
+//             }]
+//         }
+//     }
+// }); // Random Numbers
 
-/**
- * For some reason, stopping the execution of this file here makes the charts load correctly.
- * Until we figure that out, leave this in.
- * Thanks, JavaScript.
- */
-null.ignoreThisError
\ No newline at end of file
+// /**
+//  * For some reason, stopping the execution of this file here makes the charts load correctly.
+//  * Until we figure that out, leave this in.
+//  * Thanks, JavaScript.
+//  */
+// null.ignoreThisError
```

Comment:
As fun as this hack was, it does break the live view reloading, so the charts are going to have to be less impressive for a short time. They still render, just in a way that seems a little incorrect, so we can open up an issue to address that once we tackle #45